### ISSUE TITLE
Cannot change DNS Protocol

### ISSUE THREAD

**Issue by Quorum75** [AdguardTeam/AdguardForiOS#1490]
**Issue Details**

AdGuard version: 4.0.0 (367)
Device model and storage size: iPhone 11 Pro Max
Operating system and version: 13.3.1
Browser or App: AG DNS Potocol

**Expected Behavior**
Being able to change the protocol (DNSCrypt DoH...) 

**Actual Behavior**
Cannot change the protocol. It is blocked on DoH. 
When tying there is a pop up message error: “Changes Were Not Applied”

**Screenshots**
https://ibb.co/cv4z2T3

**Comment by zznosar** [comment]
It’s duplicate of this:

https://github.com/AdguardTeam/AdguardForiOS/issues/1456

I raised it a week ago. It’s a change in feature. You need to click the Select.

**Comment by Quorum75** [comment]
Thanks much appreciated 👍🏻
